experiments/Atmos_LES/risingbubble.jl

```diff
@@ -10,6 +11,10 @@ using CLIMA.Mesh.Filters
 using CLIMA.MoistThermodynamics
 using CLIMA.PlanetParameters
 using CLIMA.VariableTemplates
+using CLIMA.Courant
+
+import CLIMA.DGmethods: courant
+import CLIMA.Grids: VerticalDirection, HorizontalDirection
```

Comment:
No longer needed because of #784.

Reply:
Awesome. Any comments on whether this courant number callback should remain? I use it to perform testing, but I think it may potentially be useful for other forms of debugging and monitoring.

Reply:
I was thinking that it might be useful to add it into the driver directly, with parameters to turn it on and to specify the interval, so that experiments could use it as needed to debug?

Reply:
But I think experiment writers can best comment on its utility. @akshaysridhar, @bischtob for instance?

Reply:
I think this (or some similar form) of Courant number monitoring is very useful. Thanks for putting it in, and please leave it (or, if @kpamnany has good ideas how to streamline it, leave it in streamlined form).